src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
+import Sidebar from '../../../../help-topics'
+import Topic from '../../../../topic'
+import Desrciption from '../../../../help-description'
+import Description from '../../../../help-description';
+import {Link} from 'react-router-dom'
+
+const StyledEventGuests=styled.div`
```

Comment:
There should be a space between the `=` and the words.
`StyledEventGuests = styled.div`